packages/authorization-process/src/services/authorizationService.ts

```diff
@@ -696,8 +698,10 @@ export function authorizationServiceBuilder(
       });
 
       return {
-        JWKKey: jwkKey,
-        client: client.data,
+        key: jwkKey,
+        client: clientToApiClient(client.data, {
+          showUsers: false,
+        }),
```

Comment:
The downside of this approach is that the conversion happens here and not in the router. By the way, the jwkKey was already being converter here.